src/vscode-bicep/src/language/bicepCacheContentProvider.ts

```diff
@@ -44,24 +47,43 @@ export class BicepCacheContentProvider
   }
 
   private getBicepCacheRequest(uri: vscode.Uri) {
-    // The URIs have the format of bicep-cache:<uri-encoded bicep file path>#<uri-encoded bicep module reference>.
-    const path = decodeURIComponent(uri.path);
-    const target = decodeURIComponent(uri.fragment);
+    const [moduleReference, cachePath] = this.decodeBicepCacheUri(uri);
```

Comment:
Code improvements and change in the format of the URI used to display source code.  The URI path has to end in ".bicep" in order for vscode to display .bicep files as Bicep. That wasn't previously the case.